source/scoped-mutex-impls/src/lib.rs

```diff
@@ -0,0 +1,324 @@
+//! Mutex primitives.
+//!
+//! This module provides impls of the [`ScopedRawMutex`] trait
+//!
+//! [`ScopedRawMutex`]: crate::ScopedRawMutex
+#![cfg_attr(not(feature = "std"), no_std)]
+#![allow(clippy::new_without_default)]
+#![allow(clippy::declare_interior_mutable_const)]
+
+use core::cell::UnsafeCell;
+use core::marker::PhantomData;
+use core::sync::atomic::{AtomicBool, Ordering};
+
+use scoped_mutex_traits::{ConstInit, ScopedRawMutex};
```

Comment:
very minor, but: we should probably make this `pub` so that downstream users can depend on just this crate?
```suggestion
pub use scoped_mutex_traits::{ConstInit, ScopedRawMutex};
```

Reply:
**Update**: Oh, I see that you put the impls in a separate crate and then made a `scoped-mutex` crate that publicly re-exports both. I don't know if we *need* to have two crates, instead of just putting the impls in `scoped-mutex` directly? But, it's fine to do that.

Reply:
yeah that's a better idea